### ISSUE TITLE
Accessibility -  Issues with tab key navigation in Axe Chrome extension. 

### ISSUE THREAD

**Issue by chandana7393** [dequelabs/axe-core#1774]
**Steps to reproduce:** 
1. Launch the Url "http://abcdcomputech.dequecloud.com/"
2. Launch the Axe chrome extension.
3. Navigate to all the interactive element on the extension using the tab key.
**Actual:**
1. Screen readers are mute.
![screen_reader_mute](https://user-images.githubusercontent.com/45285854/63326172-4938d100-c349-11e9-9f36-883a8074b835.gif)
2. If we click the "ANALYZE" button using enter key it is redirecting to "https://github.com/dequelabs/axe-core" Url.
![previous_link_trigger](https://user-images.githubusercontent.com/45285854/63326181-4d64ee80-c349-11e9-8679-3fa2822e156f.gif)
**Expected:**
1. Screen reader should announce properly.
2. Clicking the "ANALYZE" button should perform analyses and should throw violations.

**Tested Environment:**
Axe - 3.9.0.22715v.
Chrome - 76.0.3809.100v.
OS - Windows 10 64 bit.

CC: @padmavemulapati

**Comment by scurker** [comment]
@chandana7393 Steps 5-7 sounds like it should be a separate issue. Would you mind splitting this issue? That way 1 fix per pull request.

**Comment by scurker** [comment]
This has been difficult to narrow down because I'm not sure what change would've introduced this. The behavior I'm experiencing using Voiceover is that none of the interactive elements are read using <kbd>tab</kbd> for interactive elements, but in DOM mode with <kbd>←</kbd> and <kbd>→</kbd> keys does work as expected. I dug around in the different extensions trying to narrow things out:

* It happens with and without axe-pro
* It happens in all of the extensions (axe, attest, coconut)
* It happens with Chrome (on both Windows w/ NVDA) and MacOS
* It happens with older versions of the extension (tested with `3.8.0` released back in June)
* I tested _other_ extensions (such as Redux/React devtools) and the the behavior also happens there

The last point makes me think it's a change with Chrome itself but I'm not 100% sure yet how to identify what the change is.

**Comment by chandana7393** [comment]
Created a separate ticket for 5th and 6th step.
https://github.com/dequelabs/axe-core/issues/1779

**Comment by WilcoFiers** [comment]
I haven't been able to reproduce this issue. Either way, I think there are a few things we need to do.

1. We need to narrow down the problem. When was this introduced? If this is a browser issue, let's figure out which version of Chrome this was started.
2. Report the issue to Google
3. Document the issue and help users understand what to do to get around it.
